### PR TITLE
i fucking hate AI vox

### PR DESCRIPTION
**Pull request by ghost**
:cl: steamp0rt
add: Added an independent preference for AI VOX.
/:cl:

### CODE REVIEW

**Comment by Manly-Dwarf** [comment]
Are you for real
Bringing back the pref for vox is fine, but seriously removing the word "accelerating" because a player said "CLOWN ASS ACCELERATING" a couple times one round is just stupid. Stop dongcoding.

**Comment by ghost** [comment]
i've seen AIs that pretty much just copy/paste ACCELERATING until they hit the word limit

**Comment by inkticious** [comment]
honestly, VOX is useless because of how many words are banned, if youre gonna ban every word just remove VOX, when ever im AI i can't even annouce something i saw to the station because of how many words are banned, so i end up having to turn on the common radio, if you're so worried about people spamming job ban them instead of removing something that someone who isnt gonna spam is going to use for a good reason, and helpful reason.

**Comment by MrWankman** [comment]
> honestly, VOX is useless because of how many words are banned, if youre gonna ban every word just remove VOX, when ever im AI i can't even annouce something i saw to the station because of how many words are banned, so i end up having to turn on the common radio, if you're so worried about people spamming job ban them instead of removing something that someone who isnt gonna spam is going to use for a good reason, and helpful reason.

I don't think words are banned, so much that words are just limited because the VOX's audio is only the intro audio to Half Life and such is not very robust for replicating most of the language. It is however a good meme.

**Comment by inkticious** [comment]
good meme is good

**Comment by Carbonhell** [comment]
+1 for the pref, -1 for the word removal. Just turn off your prefs if someone is fucking with words that trigger you.

**Comment by ghost** [comment]
ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING 
ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING 
ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING 
ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING ACCELERATING
WOOP

**Comment by GuyonBroadway** [comment]
CLOWN ACCELERATING 20 KILO METER JONSON INTO CAPTAIN ASS HOLE

**Comment by ghost** [comment]
ACCELERATING has been re-added

**Comment by Carbonhell** [comment]
Make it start enabled. It should be opt-out, not opt-in.

**Comment by ghost** [comment]
Done

**Comment by praisenarsie** [comment]
image

**Comment by praisenarsie** [comment]
also i dislike the lack of use of the dongcode label

File: hippiestation/code/modules/client/preferences_toggles.dm [hunk 1]
```diff
@@ -25,4 +25,19 @@ TOGGLE_CHECKBOX(/datum/verbs/menu/Settings/Sound, toggle_footsteps)()
...
+	set name = "Hear/Silence AI VOX"
+	set category = "Preferences"
+	set desc = "Hear AI VOX"
+	usr.client.prefs.hippie_toggles ^= SOUND_VOX
+	usr.client.prefs.save_preferences()
+	if(usr.client.prefs.hippie_toggles & SOUND_VOX)
+		to_chat(usr, "You will now hear AI VOX.")
+	else
+		to_chat(usr, "You will no longer hear AI VOX.")
+	SSblackbox.record_feedback("nested tally", "preferences_verb", 1, list("Toggle AI VOX", "[usr.client.prefs.toggles & SOUND_VOX ? "Enabled" : "Disabled"]")) //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
```
Comment (Carbonhell) [inline hippiestation/code/modules/client/preferences_toggles.dm]: should be hippie_toggles, not toggles

Comment (Carbonhell) [reply hippiestation/code/modules/client/preferences_toggles.dm]: i bet you just copypasted this you lazy scum reeEE

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Review by Carbonhell — APPROVED** [review]

**Comment by Carbonhell** [comment]
+1

**Comment by bardofrage** [comment]
Sure, +1.

**Pull request merged**